doc/flame/examples/lib/ray_cast.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flame/collisions.dart';
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/geometry.dart';
+import 'package:flame/palette.dart';
+import 'package:flutter/material.dart';
+
+class RayCastExample extends FlameGame with HasCollisionDetection {
+  final origin = Vector2(20, 20);
+
+  final direction = Vector2(1, 0);
+  final acceleration = Vector2(0, 1);
```

Comment:
This should be named velocity or speed (it can be just a double) and then in update you should multiply this with `dt`.
The value should then be in pixels per second, so something like 60 (which is approximately what it is now) might be a fitting value, try out one that has a good speed.